src/main.rs

```diff
@@ -55,8 +54,28 @@ enum Command {
     /// account's public or private key. See the `EXAMPLES` below.
     Account(Account),
     Sign(Sign),
+    /// Present the sum of all account balances under a single wallet balance.
+    ///
+    /// Only includes accounts that have been previously derived, i.e. those
+    /// that show under `forc-wallet accounts`.
+    Balance(Balance),
+}
+
+#[derive(Debug, Args)]
+struct Balance {
+    // Account-specific args.
+    #[clap(flatten)]
+    account: account::Balance,
+    /// Show the balance for each individual non-empty account before showing
+    /// the total.
+    #[clap(long)]
+    accounts: bool,
 }
 
+/// The default network used in the case that none is specified.
+const BETA_2_URL: &str = "https://node-beta-2.fuel.network";
+const BETA_2_FAUCET_URL: &str = "https://faucet-beta-2.fuel.network";
```

Comment:
If we are not planning to add support for aliases that points to different beta networks, we might end up changing this const to point `BETA_3` once it is released and if that is the case maybe it would make sense to rename this like:

```suggestion
const DEFAULT_URL: &str = "https://node-beta-2.fuel.network";
const DEFAULT_FAUCET_URL: &str = "https://faucet-beta-2.fuel.network";
```

Reply:
Yeah at the time I was thinking of potentially adding `--beta-2` or `--beta-3` as shorthand for `--node-url ...` args, and add a `DEFAULT_URL` once we have more than one. That said I'm not sure how useful it will be to support the old beta networks like that - it might be useful to not have those flags as we'd likely want to remove support later anyways :thinking:

Reply:
I've left in the BETA_2 consts for now, mostly so that we can continue to recommend the correct faucet network as we add new networks.